games/minetest/Portfile

```diff
@@ -23,7 +24,7 @@ set game_mastersite     https://github.com/minetest/minetest_game/archive/
 
 distfiles               ${main_distfile}:main \
                         ${game_distfile}:game
-                        
+
```

Comment:
what happened to this line - there should be not tabs or spaces - just an empty line

Reply:
What is the issue here?

IMO the "tabs and spaces" were (a) deleted and be (b) came from the original Profile in first place.

Case closed, requirements fulfilled I would say. Or did I get your argument wrong?